sdk/core/azure-core/azure/core/rest/__init__.py

```diff
@@ -0,0 +1,56 @@
+# --------------------------------------------------------------------------
```

Comment:
In looking up the provisional package, from what I understand of [PEP 411](https://www.python.org/dev/peps/pep-0411/#marking-a-package-provisional) is that it's only for entire packages. Is this correct?

If it is correct, would we want a preview module? https://www.python.org/dev/peps/pep-0408/

Reply:
Instead, we should just document each thing we're adding as azure.core.rest as provisional